Interface-Signoff-Marvo.md

```diff
@@ -0,0 +1,48 @@
+# Interface Signoff
+
+## Subsystem Function
+The interface conditions any input analog voltage, analog current and digital signal then transmits it to the microcontroller. 
+## Subsystem Constraints
+1) Must be able to operate with mV voltage output ranges
+2) Must support positive DC voltage ranges: 0 to 1 V, 0 to 5 V, 1 to 5 V, 0 to 10V
+3) Must be able to operate with negative voltage ranges: -10 to 10 V
+4) Must Support current output range: 4-20 mA
+5) The PCB material used must be UL certified
+
+
+## System Schematic
+
+![Interface Schematic Mark 4 PNG](https://user-images.githubusercontent.com/118490274/219834938-37dff819-c3d3-414d-8535-df8448b19ed3.PNG)
+
+## Analysis
+
```

Comment:
1. This will be extremely sensitive at low gains. Plotting the gain vs. resistance will help visualize this. 3. What effect will this have on the sensor data? 4. Show that this will work. General: some equations might help with the analysis section, e.g. op-amp gain.

Reply:
schematic changed and relevant equations added to the analysis